cedar-lean/Cedar/Data/Map.lean

```diff
@@ -68,8 +65,8 @@ def contains {α β} [BEq α] (m : Map α β) (k : α) : Bool :=
 /-- Returns the binding for `k` in `m` or `err` if none is found. -/
 def findOrErr {α β ε} [BEq α] (m : Map α β) (k : α) (err: ε) : Except ε β :=
   match m.find? k with
-  | some v => ok v
-  | _      => error err
+  | some v => .ok v
+  | _      => .error err
```

Comment:
curious why this change?  If we don't like using `ok` and `error` without dots, we shouldn't put `open Except` at the top of most of our files.  I adopted the `open Except` because it seemed like the convention we were using before I started working on the Lean